base/applications/regedit/lang/ro-RO.rc

```diff
@@ -31,17 +31,17 @@ Opțiuni:\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
   /R:user.dat    Locația fișierului user.dat urmează să fie modificată.\n\
                  Compatibil cu orice alt comutator. Ignorat.\n\
-  /C             Importă conținutul unui fișier de regiștri.\n\
-  /D             Șterge o cheie de regiștrii specificată.\n\
-  /E             Exportă conținutul unei cheie de regiștri specificată într-un fișier.\n\
-                 Dacă nu este nicio cheie specificată, toți regiștrii sunt exportați.\n\
+  /C             Importă conținutul unui fișier de registru.\n\
+  /D             Șterge o cheie de registru specificată.\n\
+  /E             Exportă conținutul unei chei de registru specificată într-un fișier.\n\
+                 Dacă nu este nicio cheie specificată, este exportat întregul registru.\n\
   /S             Modul silențios. Nu sunt afișate mesaje.\n\
   /V             Lansează interfața grafică (GUI) în modul avansat. Ignorat.\n\
   /?             Afișează informații și părăsește programul.\n\
-  [filename]     Locația fișierului ce conține informații despre regiștri ce urmează\n\
-                 să fie importați. Când e folosit cu [/E], această opțiune specifică\n\
-                 locația fișierului în care informațiile despre regiștri vor fi exportate.\n\
-  [reg_key]      Cheie de regiștri ce urmează să fie modificată.\n\
+  [filename]     Locația fișierului ce conține informații despre registrul ce urmează\n\
+                 să fie importat. Când e folosit cu [/E], această opțiune specifică\n\
+                 locația fișierului în care informațiile despre registru vor fi exportate.\n\
```

Comment:
These three lines are longer than 79 characters.

Reply:
I know. In both cases I try to reduce the number of characters.